sass/components/_emoticons.scss

```diff
@@ -628,7 +628,10 @@
     }
 
     .emoji-picker__preview-image {
+        left: 2px;
```

Comment:
I tweaked this a bit since I noticed the custom emoji preview is a bit misaligned currently. Here's how it looks on daily
![Screen Shot 2021-06-24 at 2 31 25 PM](https://user-images.githubusercontent.com/3277310/123314701-e8f0b880-d4f8-11eb-864b-db2d976d1219.png)
and here's my local server
![Screen Shot 2021-06-24 at 2 31 39 PM](https://user-images.githubusercontent.com/3277310/123314722-ec843f80-d4f8-11eb-8d64-4b600784bc36.png)